src/Symfony/Component/Console/Application.php

```diff
@@ -1067,6 +1070,11 @@ public function setDefaultCommand($commandName)
         $this->defaultCommand = $commandName;
     }
 
+    /**
+     * @param string $string
+     * 
+     * @return int
+     */
```

Comment:
happy to remove, but it does add value as it provides the type info, but fair enough :)